recipes/vapor/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "vapor" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "6d7818c6556d24663ffa72352a887f3e236f10bff4559bd86e4081c7cccbe8af" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/connor-lab/vapor/archive/refs/tags/VAPOR_1.0.2.tar.gz"
```

Comment:
you can use here the template version number e.g.

`"https://github.com/connor-lab/vapor/archive/refs/tags/VAPOR_{{ version }}.tar.gz"`